### PR TITLE
Added persistent sessions to 'How to run ACCESS-CM' page

### PR DESCRIPTION
**Pull request by atteggiani**
Updated 'How to run ACCESS-CM' page with the new persistent session workflow.
Completed removed accessdev workflow and added some instructions to try and port a suite from accessdev to persistent sessions.

### CODE REVIEW

**Comment by atteggiani** [comment]
@MartinDix 
For the review, the main sections that changed are:
[Setup ACCESS-CM persistent session](https://access-hive.org.au/pr-preview/pr-649/models/run-a-model/run-access-cm/#setup-access-cm-persistent-session) and [Port suites from accessdev](https://access-hive.org.au/pr-preview/pr-649/models/run-a-model/run-access-cm/#port-suites-from-accessdev)

File: docs/models/run-a-model/run-access-cm.md [hunk 1]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+<br>
+
+### Open the terminal in the VDI Desktop
+Once the new tab opens, you will see a Desktop with a few folders on the left.
+<br>
+To open the terminal, click on the black terminal icon at the top of the window. You should now be connected to a <i>Gadi</i> computing node.
+<img src="/assets/run_access_cm/open_are_vdi_terminal.gif" alt="Open ARE VDI terminal" class="example-img" loading="lazy"/>
+
+## Setup {{ model }} persistent session
+To support the use of long-running processes (such as ACCESS models runs), NCI provides a service on <i>Gadi</i> called <a href="https://opus.nci.org.au/display/Help/Persistent+Sessions" target="_blank">persistent sessions</a>.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:93]: Edit: “To support the use of long-running processes, such as ACCESS model runs, ..."

File: docs/models/run-a-model/run-access-cm.md [hunk 2]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+To start a new persistent session on <i>Gadi</i> (in a login node, or in an ARE terminal instance), run the following command:
+<pre><code>persistent-sessions start &lt;name&gt;</code></pre>
+
+This will start a persistent session with the given <code>name</code> that runs under your <a href="/getting_started/first_steps#change-default-project-on-gadi">default project</a>. 
+<br>
+If you want to assign a different project to the persistent session, use the option <code>-p</code>:
+<pre><code>persistent-sessions start -p &lt;project&gt; &lt;name&gt;</code></pre>
+
+<div class="note">
+    The project assigned to a persistent session does not have to be necessarily the same to the project used to run {{ model }} configuration, but needs to have allocated <i>Service Units</i> (SU).
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:107]: Edit: "While the project assigned to a persistent session does not have to be the same as the project used to run the {{ model }} configuration, it does need to have allocated <i>Service Units</i> (SU)."

File: docs/models/run-a-model/run-access-cm.md [hunk 3]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+    <br>
+    For more information, check how to <a href="/getting_started/first_steps#join-relevant-nci-projects">Join relevant NCI projects</a>.
+</div>
+<terminal-window data="input">
+    <terminal-line>persistent-sessions start &lt;name&gt;</terminal-line>
+    <terminal-line data="output">session &lt;persistent-session-uuid&gt; running - connect using</terminal-line>
+    <terminal-line data="output">&emsp;ssh &lt;name&gt;.&lt;$USER&gt;.&lt;project&gt;.ps.gadi.nci.org.au</terminal-line>
+</terminal-window>
+
+To list all the active persistent sessions run:
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:117]: Edit “To list all active persistent sessions, run:"

File: docs/models/run-a-model/run-access-cm.md [hunk 4]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+<pre><code>persistent-sessions list</code></pre>
+
+<terminal-window data="input">
+    <terminal-line>persistent-sessions list</terminal-line>
+    <terminal-line data="output">&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&ensp;UUID&emsp;&emsp;PROJECT&emsp;&ensp;&ensp;ADDRESS&emsp;&emsp;&emsp;&emsp;CPUTIME&emsp;MEMORY</terminal-line>
+    <terminal-line data="output">&lt;persistent-session-uuid&gt;&emsp;&lt;project&gt;&emsp;10.9.0.62&emsp;00:00:05.213&emsp;30.5M</terminal-line>
+</terminal-window>
+
+
+The full name of a newly created persistent session is formatted as:
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:127]: Edit: “The name of a newly-created persistent session has the following format:"

File: docs/models/run-a-model/run-access-cm.md [hunk 5]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+
+After starting the persistent session, it is essential to assign it to the {{ model }} run.
+<br>
+The easiest way to do so, is to insert the persistent session full name into the file <code>~/.persistent-sessions/cylc-session</code>.
+<br>
+You can do it manually, or by running the following command:
+
+<pre><code>cat > ~/.persistent-sessions/cylc-session <<< &lt;name&gt;.&lt;$USER&gt;.&lt;project&gt;.ps.gadi.nci.org.au</code></pre>
+
+For example, if the user <code>dm5220</code> started a persistent session named <code>cylc</code>, under the project <code>tm70</code>, the command will be:
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:141]: Is ‘dm5220' your gadi username? I would use something more generic like ‘ab1234'

File: docs/models/run-a-model/run-access-cm.md [hunk 6]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+
+<terminal-window data="input">
+    <terminal-line>cat > ~/.persistent-sessions/cylc-session <<< cylc.dm5220.tm70.ps.gadi.nci.org.au</terminal-line>
+    <terminal-line data="input" linedelay="1000">cat ~/.persistent-sessions/cylc-session</terminal-line>
+    <terminal-line data="output">cylc.dm5220.tm70.ps.gadi.nci.org.au</terminal-line>
+</terminal-window>
+
+For more information on how to specify the target session, check <a href="https://opus.nci.org.au/display/DAE/Run+Cylc7+Suites#RunCylc7Suites-SpecifyTargetSession" target="_blank">Specify Target Session on Cylc7 Suites</a>.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:149]: Edit: “target="_blank">Specify Target Session with Cylc7 Suites</a>."

File: docs/models/run-a-model/run-access-cm.md [hunk 7]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+<div class="note">
+    You can concurrently submit multiple {{ model }} runs using the same persistent session, without the need to start another one. Therefore, the process of specifying the target persistent session for {{ model }} can potentially be done only once.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:151]: Edit: “ You can simultaneously submit multiple {{ model }} runs using the same persistent session without needing to start a new one. Hence, the process of specifying the target persistent session for {{ model }} should only need to be done once."

File: docs/models/run-a-model/run-access-cm.md [hunk 8]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+            <terminal-line>module use /g/data/hr22/modulefiles</terminal-line>
+            <terminal-line>module load cylc7/23.09</terminal-line>
+            <terminal-line data="output">Using the cylc session &lt;name&gt;.&lt;$USER&gt;.&lt;project&gt;.ps.gadi.nci.org.au</terminal-line>
+            <terminal-line data="output"></terminal-line>
+            <terminal-line data="output">Loading cylc7/23.09</terminal-line>
+            <terminal-line data="output">&emsp;Loading requirement: mosrs-setup/1.0.1</terminal-line>
+        </terminal-window>
+    </li>
+    <div class="note">
+        Make sure to load a version of <i>Cylc</i> >= <code>23.09</code>, as earlier versions do not support the persistent sessions workflow.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:194]: Edit: “Make sure to load a version of <i>Cylc</i> >= <code>23.09</code> as earlier versions do not support the persistent session workflow."

File: docs/models/run-a-model/run-access-cm.md [hunk 9]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
         <br>
-        Suites are created in the user's home directory on <i>Gadi</i> under <code>~/roses/&lt;suite-ID&gt;</code>.
+        Also, before loading the <i>Cylc</i> module, make sure to have started a persistent session and to have assigned it to the {{ model }} workflow. For more information about these steps, check <a href="{{ '#setup-%s-persistent-session'%model.lower() }}">Setup {{ model }} persistent session</a>.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:196]: Edit: "Also, before loading the <i>Cylc</i> module, make sure you have started a persistent session and assigned it to the {{ model }} workflow…"

File: docs/models/run-a-model/run-access-cm.md [hunk 10]
```diff
@@ -802,64 +585,78 @@ This directory contains two subdirectories:
...
+    <terminal-line data="input">ls /scratch/&lt;$PROJECT&gt;/&lt;$USER&gt;/archive/cy339/restart/atm</terminal-line>
+    <terminal-line class="ls-output-format">cy339a.da09500201_00 cy339a.da09510101_00 cy339.xhist-09500131 cy339.xhist-09501231 </terminal-line>
+</terminal-window>
+
 Files formatted as <code>&lt;suite-name&gt;a.xhist-&lt;year&gt;&lt;month&gt;&lt;day&gt;</code> contain metadata information.
 
+## Port suites from accessdev
+<i>accessdev</i> was the server used for {{ model }} run submission workflow before the update to persistent sessions.
+<br>
+If you have a suite that was running on <i>accessdev</i>, you can run it on persistent sessions by doing the following steps:
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:617]: Edit: ”If you have a suite that was running on <i>accessdev</i>, you can run it using persistent sessions by carrying out the following steps:"

**Review by heidinett — CHANGES_REQUESTED** [review]
Looks good Davide! 
I made only some minor edit suggestions where applicable.

File: docs/models/run-a-model/run-access-cm.md (hunk 4 above)
Comment (atteggiani) [reply docs/models/run-a-model/run-access-cm.md:127]: I say "full name" to distinguish it from the persistent session `name` mentioned before.

File: docs/models/run-a-model/run-access-cm.md [hunk 11]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+
+The full name of a newly created persistent session is formatted as: 
+<br>
+<code>&lt;name&gt;.&lt;$USER&gt;.&lt;project&gt;.ps.gadi.nci.org.au</code>.
+
+### Specify {{ model }} target persistent session
+
+After starting the persistent session, it is essential to assign it to the {{ model }} run.
+<br>
+The easiest way to do so, is to insert the persistent session full name into the file <code>~/.persistent-sessions/cylc-session</code>.
```
[the remark this answers is not in the document]
Comment (atteggiani) [reply docs/models/run-a-model/run-access-cm.md:135]: Same thing here, changed "name" with "label".

File: docs/models/run-a-model/run-access-cm.md [hunk 12]
```diff
@@ -1,33 +1,11 @@
...
+# Run {{ model }}
 <div class="note">
-    The workflow to run {{ model }} is currently in transition from <i>accessdev</i> to ARE/<i>Gadi</i>.
+    <b>Important for <i>accessdev</i> users!</b>
     <br>
-    The above tabs allow you choose the type of workflow you would prefer to follow.
-    <br>
-    <br>
-    If you are new to ACCESS-CM, we <b>strongly recommend</b> that you follow the <b>ARE/<i>Gadi</i></b> workflow, as the <i>accessdev</i> workflow will soon be discontinued.
+    If you are an <i>accessdev</i> user, make sure you are a member of <a href="https://my.nci.org.au/mancini/project/hr22/join" target="_blank">hr22</a> and <a href="https://my.nci.org.au/mancini/project/ki32/join" target="_blank">ki32</a> projects, and then check the new <a href="{{ '#setup-%s-persistent-session'%model.lower() }}">persistent session worflow for {{ model }}</a>, and how to <a href="#port-suites-from-accessdev">port suites from accessdev</a>.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:6]: Edit: "If you are an accessdev user, make sure you are a member of hr22 and ki32 projects. Then, refer to instructions on {how to Setup ACCESS-CM persistent session} and how to {Port suites from accessdev}."

File: docs/models/run-a-model/run-access-cm.md [hunk 13]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
-                <br>
-                To complete these steps, refer to the <i>SSH & SSH Agent</i> section in the <a href="https://accessdev.nci.org.au/trac/wiki/GettingConnected">Getting Connected to Accessdev</i></a> guide.
-            </li>
-        </div>
-    </div>
-    <!-- End of tab content -->
+    <li>
+    <b>Connection to an ARE VDI Desktop (optional)</b>
+        <br>
+        To run {{ model }}, it is preferred to start an <a href="https://are.nci.org.au/pun/sys/dashboard/batch_connect/sys/desktop_vnc/ncigadi/session_contexts/new" target="_blank">Australian Research Environment (ARE) VDI Desktop</a> session.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:35]: Edit: “To run ACCESS-CM, start an [Australian Research Environment (ARE) VDI Desktop session."

File: docs/models/run-a-model/run-access-cm.md [hunk 14]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
-                This is a list of all project data storage (joined by <code>+</code> signs) needed for the {{ model }} simulation. In ARE, storage locations need to be explicitly defined to access data from within a VDI instance.
-                <br>
-                Every {{ model }} simulation can be unique and input data can originate from various sources. Hence, if your simulation requires data stored in project folders other than <code>access</code>, <code>hh5</code>, <code>hr22</code> or <code>ki32</code>, you need to add those projects to the storage path.
-                <br>
-                For example, if your {{ model }} simulation requires data stored in <code>/g/data/tm70</code> and <code>/scratch/w40</code>, your full storage path will be: <code>gdata/access+gdata/hh5+gdata/hr22+gdata/ki32<b>+gdata/tm70+scratch/w40</b></code>
-            </li>
-        </ul>
-        Launch the ARE session and, once it starts, click on <i>Launch VDI Desktop</i>.
-        <img src="/assets/run_access_cm/launch_are_vdi.gif" alt="Launch ARE VDI session" class="example-img" loading="lazy"/>
+## Setup ARE VDI Desktop (optional)
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:42]: Edit: "Setup an ARE VDI Desktop (optional)"

File: docs/models/run-a-model/run-access-cm.md [hunk 15]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+To skip this step and run {{ model }} from <i>Gadi</i> login node instead, check <a href="{{ '#setup-%s-persistent-session'%model.lower() }}">Setup {{ model }} persistent session</a>.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:43]: Edit: "To skip this step and instead run ACCESS-CM from a Gadi login node, refer to instructions on how to [Setup an ACCESS-CM persistent session]."

File: docs/models/run-a-model/run-access-cm.md [hunk 16]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+
+### Launch ARE VDI Session
+Go to the <a href="https://are.nci.org.au/pun/sys/dashboard/batch_connect/sys/desktop_vnc/ncigadi/session_contexts/new" target="_blank">ARE VDI</a> page and launch a session with the following directives:
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:46]: Edit: replace “directives” with “entries"

File: docs/models/run-a-model/run-access-cm.md [hunk 17]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+        <br>
+        This means that the ARE VDI session only needs to carry out setup steps as well as starting the run itself. All these tasks can be done within 2 hours.
+    </li>
+    <li>
+        <b>Queue</b> &rarr; <code>normalbw</code>
+    </li>
+    <li>
+        <b>Compute Size</b> &rarr; <code>tiny</code> (1 CPU)
+        <br>
+        As said above, the ARE VDI session is only needed for setup and startup tasks, which can be easily done with only 1 CPU.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:63]: Edit: replace “said” with “mentioned”, replace “done with “accomplished” and remove “only” (already used only is the sentence)

File: docs/models/run-a-model/run-access-cm.md [hunk 18]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+<br>
+
+### Open the terminal in the VDI Desktop
+Once the new tab opens, you will see a Desktop with a few folders on the left.
+<br>
+To open the terminal, click on the black terminal icon at the top of the window. You should now be connected to a <i>Gadi</i> computing node.
+<img src="/assets/run_access_cm/open_are_vdi_terminal.gif" alt="Open ARE VDI terminal" class="example-img" loading="lazy"/>
+
+## Setup {{ model }} persistent session
+To support the use of long-running processes, such as ACCESS models runs, NCI provides a service on <i>Gadi</i> called <a href="https://opus.nci.org.au/display/Help/Persistent+Sessions" target="_blank">persistent sessions</a>.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:93]: Edit: replace “models” with “model"

File: docs/models/run-a-model/run-access-cm.md [hunk 19]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+
+To run {{ model }}, you need to start a persistent session and set it as the target session for the model run.
+
+### Start a new persistent session
+To start a new persistent session on <i>Gadi</i> (in a login node, or in an ARE terminal instance), run the following command:
+<pre><code>persistent-sessions start &lt;name&gt;</code></pre>
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:99]: Edit: “To start a new persistent session on Gadi, using either a login node or an ARE terminal instance, run the following command:"

**Comment by atteggiani** [comment]
Thank you @heidinett 
Updated edits. 

Merging now.

File: docs/models/run-a-model/run-access-cm.md [hunk 20]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+
+This will start a persistent session with the given <code>name</code> that runs under your <a href="/getting_started/first_steps#change-default-project-on-gadi">default project</a>.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:103]: Edit: replace “the given name” with “a given name"

File: docs/models/run-a-model/run-access-cm.md [hunk 21]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+
+The label of a newly-created persistent session has the following format: 
+<br>
+<code>&lt;name&gt;.&lt;$USER&gt;.&lt;project&gt;.ps.gadi.nci.org.au</code>.
+
+### Specify {{ model }} target persistent session
+
+After starting the persistent session, it is essential to assign it to the {{ model }} run.
+<br>
+The easiest way to do this, is to insert the persistent session label into the file <code>~/.persistent-sessions/cylc-session</code>.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:137]: Edit: remove comma "The easiest way to do this is to…"

File: docs/models/run-a-model/run-access-cm.md [hunk 22]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+<br>
+You can do it manually, or by running the following command:
+
+<pre><code>cat > ~/.persistent-sessions/cylc-session <<< &lt;name&gt;.&lt;$USER&gt;.&lt;project&gt;.ps.gadi.nci.org.au</code></pre>
+
+For example, if the user <code>abc123</code> started a persistent session named <code>cylc</code>, under the project <code>xy00</code>, the command will be:
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:143]: Edit: remove comma “… a persistent session named cylc under the project…"

File: docs/models/run-a-model/run-access-cm.md (hunk 20 above)
Comment (atteggiani) [reply docs/models/run-a-model/run-access-cm.md:103]: Why "a" given name?
That's the specific `name` it was used in the command, I think in this case is correct "the" given name.

File: docs/models/run-a-model/run-access-cm.md [hunk 23]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+
+<terminal-window data="input">
+    <terminal-line>cat > ~/.persistent-sessions/cylc-session <<< cylc.abc123.xy00.ps.gadi.nci.org.au</terminal-line>
+    <terminal-line data="input" linedelay="1000">cat ~/.persistent-sessions/cylc-session</terminal-line>
+    <terminal-line data="output">cylc.abc123.xy00.ps.gadi.nci.org.au</terminal-line>
+</terminal-window>
+
+For more information on how to specify the target session, check <a href="https://opus.nci.org.au/display/DAE/Run+Cylc7+Suites#RunCylc7Suites-SpecifyTargetSession" target="_blank">Specify Target Session with Cylc7 Suites</a>.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:151]: Edit: replace “check” with “read”.

Comment (atteggiani) [reply docs/models/run-a-model/run-access-cm.md:151]: I will replace it with "refer to"

File: docs/models/run-a-model/run-access-cm.md [hunk 24]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+<div class="note">
+    You can simultaneously submit multiple {{ model }} runs using the same persistent session without needing to start a new one. Hence, the process of specifying the target persistent session for {{ model }} should only need to be done once.
+    <br>
+    After specifying the {{ model }} target persistent session the first time, to run {{ model }} you just have to make sure to have an active persistent session named like the {{ model }} target persistent session.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:155]: Edit: “... the first time, to run {{ model }} you just need to make sure that you have an active persistent session…"

File: docs/models/run-a-model/run-access-cm.md [hunk 25]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
-    </div>
-    <!-- accessdev -->
-    <div>
-        For this example you can use <code>u-br565</code>, which is the CMIP6-release preindustrial experiment suite.
-        <br>
-        Typically, an existing suite is copied and then edited as needed for a particular run.
-    </div>
-</div>
-<!-- End of tab content -->
+For this example you can use <code>u-cy339</code>, which is a preindustrial experiment suite.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:171]: Edit: replace “preindustrial” with “pre-industrial” (I prefer the hyphen, but I can be overruled)

File: docs/models/run-a-model/run-access-cm.md [hunk 26]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+            <terminal-line>module use /g/data/hr22/modulefiles</terminal-line>
+            <terminal-line>module load cylc7/23.09</terminal-line>
+            <terminal-line data="output">Using the cylc session &lt;name&gt;.&lt;$USER&gt;.&lt;project&gt;.ps.gadi.nci.org.au</terminal-line>
+            <terminal-line data="output"></terminal-line>
+            <terminal-line data="output">Loading cylc7/23.09</terminal-line>
+            <terminal-line data="output">&emsp;Loading requirement: mosrs-setup/1.0.1</terminal-line>
+        </terminal-window>
+    </li>
+    <div class="note">
+        Make sure to load a version of <i>Cylc</i> >= <code>23.09</code> as earlier versions do not support the persistent sessions workflow.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:196]: Edit: "Make sure to load a Cylc version of 23.09 (or later) as earlier versions…"

File: docs/models/run-a-model/run-access-cm.md [hunk 27]
```diff
@@ -51,262 +29,224 @@ If you are unsure whether {{ model }} is the right choice for your experiment, t
...
+        <br>
+        Also, before loading the <i>Cylc</i> module, make sure to have started a persistent session and assigned it to the {{ model }} workflow. For more information about these steps, check <a href="{{ '#setup-%s-persistent-session'%model.lower() }}">Setup {{ model }} persistent session</a>.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md]: I would replace “check” with either “see” or “read"

Comment (atteggiani) [reply docs/models/run-a-model/run-access-cm.md]: Replacing with "refer to"

File: docs/models/run-a-model/run-access-cm.md [hunk 28]
```diff
@@ -802,64 +585,78 @@ This directory contains two subdirectories:
...
+<br>
+If you have a suite that was running on accessdev, you can run it using persistent sessions by carrying out the following steps:
+
+<ol>
+    <li>
+        <b>Initialisation step</b>
+        <br>
+        To set the correct SSH configuration for <i>Cylc</i>, some SSH keys need to be created in the <code>~/.ssh</code> directory.
+        <br>
+        To create the needed SSH keys, run the following command:
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:627]: replace “needed” with “required” (already used “need” in this sentence)

File: docs/models/run-a-model/run-access-cm.md [hunk 29]
```diff
@@ -802,64 +585,78 @@ This directory contains two subdirectories:
...
+        <div class="note">
+            You only need to run this initialisation step once.
+        </div>
+    </li>
+    <li>
+        <b>Set host to localhost</b>
+        <br>
+        To enable <i>Cylc</i> to submit PBS jobs directly from the persistent session, the suite configuration should have its <code>host</code> set as <code>localhost</code>.
+        <br>
+        You can manually set all occurrencies of <code>host</code> to <code>localhost</code> in the suite configuration files.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:638]: replace “occurrencies” with “occurrences”.

File: docs/models/run-a-model/run-access-cm.md [hunk 30]
```diff
@@ -802,64 +585,78 @@ This directory contains two subdirectories:
...
+        <br>
+        Alternatively, you can run the following command in the suite folder:
+        <pre><code>grep -rl --exclude-dir=.svn "host\s*=" . | xargs sed -i 's/\(host\s*=\s*\).*/\1localhost/g'</code></pre>
+    </li>
+    <li>
+        <b>Add gdata/hr22 and gdata/ki32 in the PBS storage directives</b>
+        <br>
+        The persistent sessions workflow uses files in the <code>hr22</code> and <code>ki32</code> project folders on <i>Gadi</i>. 
+        <br>
+        Therefore, the respective folders need to be added to the <code>storage</code> directive in the suite configuration files.
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:648]: Combine above two sentences: “As the persistent session workflow uses files in the hr22 and ki32 project folders on Gadi, the respective folders need to be added to the storage directive in the suite configuration files."

File: docs/models/run-a-model/run-access-cm.md [hunk 31]
```diff
@@ -802,64 +585,78 @@ This directory contains two subdirectories:
...
+        <br>
+        You can do this manually, or run the following command in the suite folder:
```
Comment (heidinett) [inline docs/models/run-a-model/run-access-cm.md:650]: Edit: “… or run the following command from within the suite directory:"

**Review by heidinett — APPROVED** [review]
Looks good Davide! I left a few suggested edits. Feel free to accept or reject these.

File: docs/models/run-a-model/run-access-cm.md (hunk 20 above)
Comment (heidinett) [reply docs/models/run-a-model/run-access-cm.md:103]: ok, I agree with “the”.

**Comment by atteggiani** [comment]
Thank you @heidinett. 

Merging now.

**Pull request merged**